execute_pipeline.py

```diff
@@ -145,6 +145,8 @@ def _load_config_spec(config_spec, config_sections, repl_vars):
         all_config_data = yaml.load(config_file)
     for section in config_sections:
         data.update(all_config_data[section])
+    if language in all_config_data:
```

Comment:
It is idempotent, so no behaviour problems.

I can't think of a case where someone would want to leave out the language section, but it seems desirable to make the command line args capable of fully specifying what you want to load.